### DISCUSSION TITLE
Right click to stop the UI from toggling between the graph and timer

### DISCUSSION THREAD

**Discussion by SashaCanfield03** [ppy/osu#29081] [Ideas]
I think it's really annoying that it keeps toggling back and forth between the graph and the timer. I think there should be a way to stop it from doing that, like right-clicking it, and making the little oval-thing gold or a different shade of pink to indicate that it will stop swapping the UI.

If I could still submit scores on a modified client I would just set the timer to an hour to make it go away, but now I have to ask for it here! Annoying!

**Comment by SupDos** [comment]
I believe they plan on allowing locking the page as per https://github.com/orgs/ppy/projects/18/views/1?pane=issue&itemId=72207176

**Comment by bdach** [comment]
as stated above this is a frequent point of feedback and will be addressed but i can *not* help myself from dunking on the following "criticism":

> If I could still submit scores on a modified client I would just set the timer to an hour to make it go away

if you could submit scores on a modified client there would be nothing stopping you from giving yourself seven bajillion points for every score which seems pretty undesirable for a game with leaderboards wouldn't you say
